src/main/util.ts

```diff
@@ -120,14 +125,21 @@ const getMetadataForVideo = (video: any) => {
     }
 }
 
+const saveMetadataForVideo = (videoPath: string, metadata: any) => {
+    const metadataFile = getMetadataFileForVideo(videoPath);
+
```

Comment:
random new line

Reply:
It gets the filename and path for the metadata for the given video fil in order to save the metadata. Before, a few places that saved or read metadata would first get filename, the call the appropriate function to save/load it.

Now, the method that saves/loads will determine the filename based on the video path. See also `getMetadataForVideo()`, `writeMetadataFile()`, and `deleteVideo()` where it's used as well.